src/ai/backend/manager/api/auth.py

```diff
@@ -375,6 +377,27 @@ async def sign_request(sign_method: str, request: web.Request, secret_key: str)
         raise InvalidAuthParameters(e.args[0])
 
 
+def validate_ip(request: web.Request, user: Mapping[str, Any]):
+    allowed_client_ip = user.get("allowed_client_ip", None)
+    if not allowed_client_ip or allowed_client_ip is None:
+        # allowed_client_ip is None or [] - empty list
+        return
+    assert isinstance(allowed_client_ip, list)
+    raw_client_addr: str | None = (
+        request.headers.get("X-BackendAI-IP")
+        or request.headers.get("X-Forwarded-For")
+        or request.remote
+    )
```

Comment:
Same here. I think we won't need `X-BackendAI-IP` header anymore.
```suggestion
        request.headers.get("X-Forwarded-For")
        or request.remote
    )
```